app/models/user.rb

```diff
@@ -8,6 +8,7 @@ class User < ApplicationRecord
 
   # Relations
   has_many :rents, dependent: :destroy
+  has_many :book_suggestions, dependent: :destroy
```

Comment:
In another context, a library could maybe keep the suggestions made by old users, event if they are deleted from the db. Here is ok!

Reply:
Good